oak-run/src/main/js/oak-mongo.js

```diff
@@ -433,6 +437,9 @@ var oak = (function(global){
                 unset["_collisions." + r] = "";
                 num++;
             }
+            if (num >= limit) {
+                break;
```

Comment:
Hmm, not sure this is useful. The default limit is very high an unlikely to be hit. When you give it a limit, then you know it may not clean up everything and you have to run the command again. If there is nothing to clean up then there is a message that says: "No collisions found for clusterId X".

Maybe rather a log message in any case that tells how many were removed?

Reply:
> Maybe rather a log message in any case that tells how many were removed?

+1, sounds like a good option